### DISCUSSION TITLE
Giscus support

### DISCUSSION THREAD

**Discussion by SageofPHY6Paths** [MrGreensWorkshop/MrGreen-JekyllTheme#165] [Feature requests] [answered]
### Is this enhancement or feature is related to a problem?

Thanks for making this awesome theme and making it open source. All the themes I could find were very minimalist.

https://giscus.app/ or https://github.com/giscus/giscus is a recent alternative to discus. It is free, open source, privacy focused and uses GitHub discussions.

Add giscus support to this theme.

### Describe the solution you'd like

I can add it very easily using the steps at https://giscus.app/ but I myself can't do fine changes like configure such that when the dark toggle is clicked the giscus comments also change the theme. But I guess you will be able to do this. So official support is better. by default you can keep disqus but give the option to change to giscus in the config file or somewhere.

Thanks.

**Comment by SageofPHY6Paths** [comment]
There is a small problem here though, giscus as of now doesn't have the ability to show the number of comments outside the post page. https://github.com/giscus/giscus/issues/145

But that feature is not that important in my opinion. You can set up if else functions such that in the config file if giscus is selected then the number of comments will be automatically removed.

**Comment by MrGreensWorkshop** [comment]
I definitely want to add giscus. Since the comment count feature can be disabled, I don't think it's going to be a problem.

As you mentioned, adding it is simple at first, but adapting to different screen sizes, making it compatible with browsers, and adding the dark mode feature are all time-consuming tasks. 

No promises, but I really want to add.

**Comment by MrGreensWorkshop** [comment, marked as answer]
I just added giscus, enjoy!

https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/releases/tag/v1.2.4
